docs/sdv_theme/_templates/docs-navbar.html

```diff
@@ -13,12 +13,7 @@
 
     <div id="navbar-menu" class="col-lg-9 collapse navbar-collapse">
       <ul id="navbar-main-elements" class="navbar-nav mr-auto">
-        {% set nav = get_nav_object(maxdepth=1, collapse=True) %}
-        {% for main_nav_item in nav %}
-        <li class="nav-item {% if main_nav_item.active%}active{% endif %}">
-            <a class="nav-link" href="{{ main_nav_item.url }}">{{ main_nav_item.title }}</a>
-        </li>
-        {% endfor %}
+        {{ toctree(maxdepth=1, collapse=True, includehidden=True, titles_only=True) }}
```

Comment:
get_nav_object no longer exists so I had to update this so the build would go through